server/nodeServer.js

```diff
@@ -55,25 +54,25 @@ const server = http.createServer((req, res) => {
     fs.createReadStream('./client/dist/productionBundle.js').pipe(res);
   } else if (method === 'GET' && url === '/') {
     res.end(Html(renderComponent(Reservation, urlId), 'Silverspoon', urlId));
+    fs.createReadStream('./client/dist/productionBundle.js').pipe(res);
   } else if (method === 'POST' && url === '/reservations') {
     let body = [];
     req.on('error', err => console.error(err));
     req.on('data', (chunk) => {
       body.push(chunk);
     }).on('end', () => {
       body = Buffer.concat(body).toString();
```

Comment:
I think you might be able to add the chunks to an empty string, and that should be just as good as making an array, concating it, and converting it to a string. It might be more efficient, but I need to review it God willing.